### PR TITLE
Add QR code to download app to confirmation page

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Implement download app qr code on confirmation page.

Fix small syntax issues.

## Justify why they are needed

The QR code points to staging deployment now but we should update that once we have the production deployment.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1550** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1550" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1550?utm_source=stack-comment).

File: apps/store/src/components/ConfirmationPage/ConfirmationPage.tsx [hunk 1]
```diff
@@ -21,19 +23,12 @@ export const ConfirmationPage = (props: Props) => {
...
   ) : (
-    <SpaceFlex space={0.5}>
-      <Link href={appStoreLinks.apple} passHref>
-        <AppStoreBadge type="apple" locale={locale} />
-      </Link>
-      <Link href={appStoreLinks.google} passHref>
-        <AppStoreBadge type="google" locale={locale} />
-      </Link>
-    </SpaceFlex>
+    <Image src={qrCodeImage} alt="Scan to download the app" width={128} height={128} />
```
Comment (alebedev) [inline apps/store/src/components/ConfirmationPage/ConfirmationPage.tsx:31]: Do we want to localize alt text?

**Review by alebedev — APPROVED** [review]
Looks good

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: EZFlTXgTbkrUk696F3fM.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1550)

**Review by KarnellSchultz — APPROVED** [review]

**Pull request merged**